plugins/playlist/src/components/PlaylistPage/PlaylistHeader.tsx

```diff
@@ -166,7 +170,7 @@ export const PlaylistHeader = ({ playlist, onUpdate }: PlaylistHeaderProps) => {
           },
           {
             label: `Delete ${singularTitle}`,
-            icon: <DeleteIcon />,
+            icon: <DeleteIcon className={classes.deleteicon} />,
```

Comment:
What do you think of setting color this way instead? It would be consistent with how we're coloring the delete button in the confirmation dialog [here](https://github.com/backstage/backstage/blob/06e50f7381acb3a8a849d455ef24672f43b69db7/plugins/playlist/src/components/PlaylistPage/PlaylistHeader.tsx#L204)
```suggestion
            icon: <DeleteIcon color="secondary" />,
```